test/jdk/javax/swing/JColorChooser/Test4887836.java

```diff
@@ -40,14 +40,16 @@ public class Test4887836 {
     public static void main(String[] args) throws Exception {
         String instructions = """
                                 If you do not see white area under the \"Swatches\" tab,
-                                then test passed, otherwise it failed.""";
+                                then test passed, otherwise it failed.
+
+                                NOTE: There is no Swatches tab for GTK Look and Feel, Press Pass.""";
```

Comment:
I guess better to restrict programmatically 
"GTK".equals(UIManager.getLookAndFeel().getName())
Also, add a comment why it is not applicable..

Reply:
Whatever L&F you are running on irrespective of manually or automated, it should be able to find out via getLookAndFeel..

Reply:
Yeah, you are right. Test updated to throw `jtreg.SkippedException` if it is run for GTK L&F.